br/backup-and-restore-storages.md

```diff
@@ -1,247 +1,229 @@
-2. Content of the file on the tool node at `~/.config/gcloud/application_default_credentials.json`
-3. When running in GCE or GAE, the credentials fetched from the metadata server.
+```shell
+./br restore db --db test -u "${PD_IP}:2379" \
+--storage "azure://external/backup-20220915account-name=${account name}&account-key=${account key}"
+```
 
-### Azblob URL parameters
+</div>
+</SimpleTab>
 
-| URL parameter | Description |
-|:----------|:-----|
-| `account-name` | The account name of the storage |
-| `account-key` | The access key|
-| `access-tier` | Access tier of the uploaded objects (for example, `Hot`, `Cool`, `Archive`). If `access-tier` is not set (the value is empty), the value is `Hot` by default. |
+## Authentication
 
-To ensure that TiKV and BR use the same storage account, BR determines the value of `account-name`. That is, `send-credentials-to-tikv = true` is set by default. BR infers these keys from the environment in the following order:
+When storing backup data in a cloud storage system, you need to configure authentication parameters depending on the specific cloud service provider. This section describes the authentication methods used by Amazon S3, GCS, and Azure Blob Storage, and how to configure the accounts used to access the corresponding storage service.
 
-1. If both `account-name` **and** `account-key` are specified, the key specified by this parameter is used.
-2. If `account-key` is not specified, BR tries to read the related credentials from environment variables on the node of BR. BR reads `$AZURE_CLIENT_ID`, `$AZURE_TENANT_ID`, and `$AZURE_CLIENT_SECRET` first. At the same time, BR allows TiKV to read these three environment variables from the respective nodes and access the variables using Azure AD (Azure Active Directory).
-3. If the preceding three environment variables are not configured in the BR node, BR tries to read `$AZURE_STORAGE_KEY` using an access key.
+<SimpleTab groupId="storage">
+<div label="Amazon S3" value="amazon">
 
-> **Note:**
->
-> - When using Azure Blob Storage as the external storage, you should set `send-credentials-to-tikv = true` (which is set by default). Otherwise, the backup task will fail.
-> - `$AZURE_CLIENT_ID`, `$AZURE_TENANT_ID`, and `$AZURE_CLIENT_SECRET` respectively refer to the application ID `client_id`, the tenant ID `tenant_id`, and the client password `client_secret` of the Azure application. For details about how to confirm the presence of the three environment variables, or how to configure the environment variables as parameters, see [Configure environment variables](/br/backup-storage-azblob.md#configure-environment-variables).
+Before backup, configure the following privileges to access the backup directory on S3.
 
-## Command-line parameters
+- Minimum privileges for TiKV and `br` to access the backup directories during backup: `s3:ListBucket`, `s3:PutObject`, and `s3:AbortMultipartUpload`
```

Comment:
I think we need to change `br` to BR, these permission requirements are the same for all forms of BR(br cli, tidb operator backup)

Reply:
Fixed in [ef1a507](https://github.com/pingcap/docs/pull/11300/commits/ef1a507558228dc0cad228751a57fddfc4b1e977)